tests/unit-tests/UnitTests/Extensions/DictionaryExtensionsTests.cs

```diff
@@ -8,6 +8,24 @@ namespace form_builder_tests.UnitTests.Extensions
 {
     public class DictionaryExtensionsTests
     {
+        [Fact]
```

Comment:
change to be a theory and pass in the char, to remove the forloop and make it clearer its testing multiple chars